src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunctionComparer.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.PowerFx.Core.Functions
+{
+    internal class TexlFunctionComparer : IEqualityComparer<TexlFunction>
+    {
+        public bool Equals(TexlFunction x, TexlFunction y)
+        {
+            return x.AffectsAliases == y.AffectsAliases &&
```

Comment:
This is too long, and if we add more fields it would mean we have to update this code. Two TexlFunctions are the same if the unique runtime name is the same (however this is a PA client concept and may be removed in the future). Alternatively, two functions are the same if 1) the name is the same and 2) the arg list is the same.

Reply:
:)
Done.